### PR TITLE
Update network_timezones.txt

### PR DESCRIPTION
**Pull request by danyboy666**
Added networks TV5, Moi & cie and Canal Vie.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [ ] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sb_network_timezones/network_timezones.txt [hunk 1]
```diff
@@ -1468,7 +1470,6 @@ QVC Style (UK):Europe/London
 QVC:US/Eastern
 Racing UK:Europe/London
 Radio Bremen:Europe/Berlin
-Radio Canada:Canada/Eastern
```
Comment (miigotu) [inline sb_network_timezones/network_timezones.txt]: Why did you remove this?

**Comment by danyboy666** [comment]
It's a duplicate of Radio-Canada. Or maybe there was a reason for it to be there without the "-" ?

**Comment by miigotu** [comment]
Yes, there is a reason it was there. Case and punctuation matters.

**Comment by danyboy666** [comment]
I understand.
